azure-pipelines/compliance.yml

```diff
@@ -27,4 +27,8 @@ steps:
     FileDirPath: '$(Build.StagingDirectory)'
     TreatSignatureUpdateFailureAs: 'Warning'
     SignatureFreshness: 'UpToDate'
-    TreatStaleSignatureAs: 'Error'
\ No newline at end of file
+    TreatStaleSignatureAs: 'Error'
+
+- task: CredScan@3
+  inputs:
+    scanFolder: '$(Build.SourcesDirectory)'

```

Comment:
nit: redundant
// according to [docs](https://eng.ms/docs/microsoft-security/security/azure-security/cloudai-security-fundamentals-engineering/security-integration/guardian-wiki/sdl-azdo-extension/credscan-azure-devops-build-task) that's the default value

Reply:
Thanks. Keeping for clarity that the scan requires only sources (not build).